pulley/src/interp.rs

```diff
@@ -951,6 +951,30 @@ impl OpVisitor for InterpreterVisitor<'_> {
         Continuation::Continue
     }
 
+    fn load32_u_offset64(&mut self, dst: XReg, ptr: XReg, offset: i64) -> Self::Return {
+        let val = unsafe {
+            self.state
+                .x(ptr)
+                .get_ptr::<u32>()
+                .byte_offset(offset as isize)
+                .read_unaligned()
+        };
+        self.state.x_mut(dst).set_u64(u64::from(val));
+        Continuation::Continue
+    }
```

Comment:
Would it perhaps make sense to add a generlized helper `load32_u_offset` which takes `offset: usize` which no-offset/offset8/offset64 all call? (and would presumably get inlined and not actually matter at runtime)

Reply:
Mayyybe. Right now some of the `offset`-to-`isize` conversions are `.into()` and some are `as isize` which is a small hiccup, but probably inconsequential. All said, I think I'd prefer to just leave it as-is for now and see what the future brings.